Lib/multiprocessing/shared_memory.py

```diff
@@ -563,11 +714,53 @@ def get_server(self):
         'Better than monkeypatching for now; merge into Server ultimately'
         if self._state.value != State.INITIAL:
             if self._state.value == State.STARTED:
-                raise ProcessError("Already started server")
+                raise ProcessError("Already started SharedMemoryServer")
             elif self._state.value == State.SHUTDOWN:
-                raise ProcessError("Manager has shut down")
+                raise ProcessError("SharedMemoryManager has shut down")
             else:
                 raise ProcessError(
                     "Unknown state {!r}".format(self._state.value))
-        return _Server(self._registry, self._address,
-                       self._authkey, self._serializer)
+        return self._Server(self._registry, self._address,
+                            self._authkey, self._serializer)
+
+    def SharedMemory(self, size):
+        """Returns a new SharedMemory instance with the specified size in
+        bytes, to be tracked by the manager."""
+        conn = self._Client(self._address, authkey=self._authkey)
+        try:
+            sms = SharedMemory(None, flags=O_CREX, size=size)
+            try:
+                dispatch(conn, None, 'track_segment', (sms.name,))
+            except BaseException as e:
+                sms.unlink()
+                raise e
+        finally:
+            conn.close()
```

Comment:
Could use a `with` statement (`with self.Client(...) as client:`).

Reply:
Nice! I did not think conn had a context manager in all cases but it does -- cool.  This and its near-mirror in your next comment are now changed in commit 0be05318219ca1872f07af45404aca5619c3f0d5.